tools/dds/dds-server/rs-dds-server.cpp

```diff
@@ -27,8 +53,20 @@ try
     cmd.add( debug_arg );
     cmd.parse( argc, argv );
 
+    // Intercept DDS messages and redirect them to our own logging mechanism
+    std::unique_ptr< eprosima::fastdds::dds::LogConsumer > consumer( new log_consumer() );
```

Comment:
Who is releasing this memory? We don't care cause it's on application close flow?